### PR TITLE
Bump deps

### PR DESCRIPTION
**Pull request by lishaduck**
Follow up to #255, bumps everything else.

---

I also had this commit, but npm dedupe reverted it, so I did too:

```
chore(deps): bump elm-solve-deps-wasm

Manually edited the `package-lock.json` to force `elm-solve-deps-wasm` v2.
Then, regenerated the lockfile to give `elm-test` the old version.
Workaround for rtfeldman/node-test-runner#636.
```

Ref: rtfeldman/node-test-runner#636

### CODE REVIEW

**Comment by lishaduck** [comment]
@jfmengels, can you take a look? I think all of the snapshot changes are correct, but I just skimmed 'em. The json was particularly intimidating.

**Comment by lishaduck** [comment]
@jfmengels, ping.
I worked around rtfeldman/node-test-runner#636 and then updated snapshots again.

**Comment by lishaduck** [comment]
> I **think** all of the remaining test failures are snapshots that broke when you added fixes or contained version numbers. [emphasis added]

Figured out why there were a bunch of behavioral changes: `NoUnused.Modules` got folded into `NoUnused.Exports`, which made the diff significantly larger. Removing `NoUnused.Modules` made it much more reviewable.

File: package.json [hunk 1]
```diff
@@ -49,7 +49,7 @@
     "elm-tests": "(cd template/ && elm make src/Elm/Review/Main.elm --output=/dev/null && elm-test)",
     "eslint-check": "eslint . --report-unused-disable-directives --max-warnings=0",
     "eslint-fix": "npm run eslint-check -- --fix",
-    "jest": "npx jest",
+    "jest": "jest",
```
Comment (lishaduck) [inline package.json:52]: Wrong commit, but doesn't make any behavioral change beyond shaving a few milliseconds off testing.

**Comment by jfmengels** [comment]
@lishaduck Thank you for all your work! And I'm so sorry for not tending to your pull requests :disappointed: 
I'll try to look at the rest of the PRs quickly.

**Comment by lishaduck** [comment]
> @lishaduck Thank you for all your work! And I'm so sorry for not tending to your pull requests 😞 I'll try to look at the rest of the PRs quickly.

Give me a minute before merging #264, I'm working on it literally right now.

**Pull request merged**